androidApp/src/main/java/co/nimblehq/avishek/phong/kmmic/android/ui/screen/surveydetail/SurveyDetailScreen.kt

```diff
@@ -30,12 +29,14 @@ const val ImageScaleAnimationDurationInMillis = 700
 fun SurveyDetailScreen(
     homeViewModel: HomeViewModel = getViewModel(),
```

Comment:
Why are we getting `HomeviewModel` in `SurveyDetailScreen`? We can get the survey detail information from the API, can't we? 💭

Reply:
@kaungkhantsoe Fetching the survey has some latency. But clicking the survey on the Home screen magnifies the image and displays the survey title and description immediately on the Details screen. So the screens share the same VM to use the already fetched data.